### PR TITLE
 Fix Python 3.6 f-string compatibility and condense documentation for FNO classes

### PR DESCRIPTION
**Pull request by m4e7**
There are several instances of `f"{x=}"` in the repo, which causes issues when trying to import the library in to Python 3.6. Drop the `=` in these f-string to fix this problem.

Additionally, the repeated docstrings between base class `FNO` and subclasses `FNOnd` make maintaining documentation more work than it needs to be. The parameters are used identically in the base- and sub-class, and in some cases the subclass documentation is already out of date. Delete the repeated parameter comments in favor of referring users back up to the full documentation of `FNO` (tested in a local `make` of `sphinx-gallery`).

Lastly, run `black` on touched files.

Reviewer(s): @zongyi-li @JeanKossaifi

### CODE REVIEW

**Comment by JeanKossaifi** [comment]
I don't think we need to abandon new features like f-strings that make the code easier to read just to support Python 3.6: it was released in 2016 and has reached [end of life](https://www.python.org/downloads/release/python-360/).

**Comment by JeanKossaifi** [comment]
Other changes are great!

File: neuralop/training/patching.py [hunk 1]
```diff
@@ -1,74 +1,93 @@
...
+        """Wraps a model inside a multi-grid patching"""
         super().__init__()
 
         self.skip_padding = (padding_fraction is None) or (padding_fraction <= 0)
 
-        self.levels=levels
+        self.levels = levels
 
-        if isinstance(padding_fraction, float) or isinstance(padding_fraction, int):
+        if isinstance(padding_fraction, (float, int)):
```
Comment (JeanKossaifi) [inline neuralop/training/patching.py:29]: Thanks, that's much better

File: neuralop/training/patching.py [hunk 2]
```diff
@@ -78,125 +97,169 @@ def _stitch(self, x):
...
 
-        #Size with padding removed    
+        # Size with padding removed
         size = x.size()
 
         # if self.mode == "batch-wise":
-        B = size[0]//(self.n_patches[0]*self.n_patches[1])
-        W = size[3]*self.n_patches[1]
+        B = size[0] // (self.n_patches[0] * self.n_patches[1])
+        W = size[3] * self.n_patches[1]
```
Comment (JeanKossaifi) [inline neuralop/training/patching.py:111]: I really don't like the way `black` formats operations - it actually doesn't follow pep8..

File: neuralop/training/patching.py [hunk 3]
```diff
@@ -78,125 +97,169 @@ def _stitch(self, x):
...
+                width * self.padding_fraction[1],
+            ]
+        ]
         self.padding_height = padding[0]
         self.padding_width = padding[1]
 
         patched = make_patches(x, n=2**self.levels, p=padding)
 
-        s1_patched = patched.size(-2) - 2*padding[0]
-        s2_patched = patched.size(-1) - 2*padding[1]
```
Comment (JeanKossaifi) [inline neuralop/training/patching.py:118]: Same, is there a way to force black to follow pep8 for those, or ignore them?

File: neuralop/training/patching.py [hunk 4]
```diff
@@ -78,125 +97,169 @@ def _stitch(self, x):
...
-            
+                x_sub = torch.nn.functional.pad(
+                    x_sub, pad=[s2_pad, s2_pad, 0, 0], mode="circular"
+                )
+
             if s1_pad > x_sub.size(-2):
                 diff = s1_pad - x_sub.size(-2)
-                x_sub = torch.nn.functional.pad(x_sub, pad=[0, 0, x_sub.size(-2), x_sub.size(-2)], mode='circular')
-                x_sub = torch.nn.functional.pad(x_sub, pad=[0, 0, diff, diff], mode='circular')
+                x_sub = torch.nn.functional.pad(
```
Comment (JeanKossaifi) [inline neuralop/training/patching.py:180]: That's another case where I find the formatted version actually marginally less readable

**Comment by m4e7** [comment]
> I don't think we need to abandon new features like f-strings that make the code easier to read just to support Python 3.6: it was released in 2016 and has reached [end of life](https://www.python.org/downloads/release/python-360/).

Done. I've dropped that commit.

**Comment by JeanKossaifi** [comment]
Great, thanks @m4e7, merging!

**Pull request merged**